picard/util/icontheme.py

```diff
@@ -30,8 +30,8 @@
         '/usr/share/pixmaps',
     ]
 
-if 'GNOME_DESKTOP_SESSION_ID' in os.environ:
-    _current_theme = os.popen('gconftool-2 -g /desktop/gnome/interface/icon_theme').read().strip() or None
+if 'DESKTOP_SESSION' in os.environ and os.environ['DESKTOP_SESSION'].lower() == 'gnome':
+    _current_theme = os.popen('gconftool-2 -g /desktop/gnome/interface/icon_theme').read().strip() or os.popen('gsettings get org.gnome.desktop.interface icon-theme').read().strip() or None
```

Comment:
The new command returns `'Adwaita'` on my machine - note the apostrophes. Do they need to be removed as well?